goodkey/good_key.go

```diff
@@ -89,14 +90,12 @@ func (policy *KeyPolicy) GoodKey(key crypto.PublicKey) error {
 		}
 	}
 	switch t := key.(type) {
-	case rsa.PublicKey:
-		return policy.goodKeyRSA(t)
 	case *rsa.PublicKey:
 		return policy.goodKeyRSA(*t)
```

Comment:
I think it makes sense to also update `goodKeyRSA` and `goodKeyECDSA` to take pointers instead of values, for consistency.